src/main/java/bio/terra/service/tabulardata/azure/AzureStorageTablePdao.java

```diff
@@ -32,8 +33,8 @@ private static String computeInternalLoadTag(String loadTag) {
    * Store the results of a bulk file load in an Azure Storage Table
    *
    * <p>The table name will be the result of the dataset id passed through {@link
-   * AzureStorageTablePdao#toLoadHistoryTableNameFromUUID} Entities will be partitioned on the
-   * loadTag and their row keys will be the value of {@link BulkLoadHistoryModel#getFileId()}
+   * StorageTableName#toTableName} Entities will be partitioned on the loadTag and their row keys
```

Comment:
Note that since `toTableName` is overloaded it will pick the first declared name (I think). If you want to disambiguate you can add the parameter list.